include/jemalloc/internal/tcache_structs.h

```diff
@@ -36,6 +36,11 @@ struct tcache_slow_s {
 	uint8_t		lg_fill_div[SC_NBINS];
 	/* For small bins, whether has been refilled since last GC. */
 	bool		bin_refilled[SC_NBINS];
+	/*
+	 * For small bins, the number of items we can pretend to flush before
+	 * actually flushing.
+	 */
+	uint8_t		bin_flush_delay_items[SC_NBINS];
```

Comment:
Is `uint8_t` enough for experimentation? You know, a part of me is curious about what would happen if we delay a really long time (or even forever) - like to have a sense of the extreme.

Reply:
I think so; 255 is more than the largest default cache bin size as it stands currently, so this is equivalent to more-than-one full-cache-bin flush for every item.

One of these lives in every tsd we have, so we want to be very careful about space consumption. (This was actually the motivating factor behind making this only apply for small sizes).